xinference/model/llm/pytorch/glm4v.py

```diff
@@ -84,6 +84,10 @@ def load(self):
                         f"Only 8-bit quantization is supported if it is not linux system or cuda device"
                     )
 
+        if self._check_tensorizer_integrity():
+            self._model, self._tokenizer = self._load_tensorizer()
+            return
```

Comment:
If you return here, how to run `self._save_tensorizer()` below?

Reply:
self._check_tensorizer_integrity() will check if `enable_tensorizer==True` and all the files required by the tensorizer load exist, and return directly if the conditions are met.

Reply:
If the `enable_tensorizer == False`, or checking the files are incomplete, it will be loaded through huggingface cache and serialized to tensorizer files.